docs/config/install.rst

```diff
@@ -42,12 +42,13 @@ To build **rVRRPd** from source you must have several programs and libraries ins
    to build the project and its related dependencies (crates).
  * The `OpenSSL <https://www.openssl.org/>`_ development headers
  * The `Netlink Protocol Library Suite <https://www.infradead.org/~tgr/libnl/>`_ development headers *(Linux)*
+ * The `help2man <https://www.gnu.org/software/help2man/>`_ utility, to generate manual pages.
```

Comment:
Not 100% sure how the syntax on these works with the links, but this appears correct based on the other samples above.